docs/ruleset_functions/G311_exceptions/does_zone_meet_G3_1_1d.md

```diff
@@ -0,0 +1,81 @@
+# does_zone_meet_G3_1_1d  
+
+**Schema Version:** 0.0.28
+
+**Description:** determines whether a given zone meets the G3_1_1d exception "For laboratory spaces in a building having a total laboratory 
+exhaust rate greater than 15,000 cfm, use a single system of type 5 or 7 serving only those spaces."
+
+This function gets the system-wide exhaust airflow rate following the same logic as the function **get_zone_supply_return_exhaust_relief_terminal_fan_power_dict**
+
+**Inputs:** 
+- **B_RMI**
```

Comment:
if i understand this function correctly, the determination of the exhaust airflow is happening using ONLY the P RMI. If yes, maybe we could remove B RMI from the inputs list to prevent confusion in the future?

Reply:
Can't we use B-RMI for this because we have a separate RDS checking that baseline and proposed exhausts are equal?

Reply:
we can use just B_RMI or just P_RMI.  I just removed B_RMI to rely solely on proposed, but it it just as easy to go the other way.

Reply:
discussed on call 230615 - we will leave as-is with P_RMI